test/suite/test_sweep04.py

```diff
@@ -0,0 +1,225 @@
+
+            if stressing:
+                self.session.drop(self.transient_uri(available_transient), "force")
+                available_transient += 1
+                self.create_table(self.transient_uri(created_transient))
+                created_transient += 1
+
+            rand_session = sessions[r.rand_range(0, self.nsessions)]
+
+            # In the stress part of the run, some small fraction (given by ratio_examined) will
+            # look at the transient tables.  Looking at these rarely makes them candidates for
+            # closing by the connection sweep.
+            big = 1000000   # any large number works
+            if stressing and r.rand32() % big > big * self.ratio_examined:
+                # look at transient tables
+                t = r.rand_range(available_transient, available_transient + self.transient_tables - self.transient_examined)
+                self.examine(rand_session, self.transient_uri, t, self.transient_examined)
+            else:
+                # look at a core table
+                t = r.rand_range(0, self.core_tables)
+                self.examine(rand_session, self.core_uri, t, 1)
+
+            if loopcount % 100 == 99:
+                stat_cursor = self.session.open_cursor('statistics:', None, None)
+                close = stat_cursor[stat.conn.dh_sweep_close][2]
+                remove = stat_cursor[stat.conn.dh_sweep_remove][2]
+                sweep = stat_cursor[stat.conn.dh_sweeps][2]
+                sclose = stat_cursor[stat.conn.dh_session_handles][2]
+                ssweep = stat_cursor[stat.conn.dh_session_sweeps][2]
+                tod = stat_cursor[stat.conn.dh_sweep_tod][2]
+                ref = stat_cursor[stat.conn.dh_sweep_ref][2]
+                dhandles = stat_cursor[stat.conn.dh_conn_handle_count][2]
+                files_open = stat_cursor[stat.conn.file_open][2]
+                self.pr(('DHANDLE STATS: close={}, remove={}, sweep={}, session_handles={}, '+
+                         'session_sweeps={}, sweep_tod={}, sweep_ref={}').format(
+                             close, remove, sweep, sclose, ssweep, tod, ref))
+                self.pr('  dhandle_count={}'.format(dhandles))
+                self.pr('  file_open={}'.format(files_open))
+                dhandle_counts.append(dhandles)
+                stat_cursor.close()
+                if loopcount % 100 == 999:
+                    self.conn.debug_info('handles=true')
+                self.pr('  average,slope={}'.format(str(average_slope(dhandle_counts))))
+
+            # Reset any sessions we've used. This may be necessary to trigger some session sweeps.
+            rand_session.reset()
+            self.session.reset()
+
+        # Half the run is dhandle growth, and the second half should see a decline.
+        # If everything is working right in the first half of the run, we should start
+        # to see the number of dhandles start to asymptote. So we check the slope of
```

Comment:
'asymptote' is a noun only, not a verb (according to dictionary.com).

Reply:
Yes, fixed!